.travis.yml

```diff
@@ -2,10 +2,8 @@
 language: julia
 os:
   - linux
-  - osx
 julia:
   - release
-  - nightly
```

Comment:
> Or does the Nanosoldier.jl code only run on the coordinator node with a fixed release Julia?

This. I'm never going to be running the server on a non-release version of Julia (right now it's the latest v0.5 release).